backup.py

```diff
@@ -31,128 +32,137 @@
         pass
     backupLog = open(logLoc, 'a')
     try:
-        backupLog.write("\n" + datetime.now().strftime('%Y-%m-%d %H:%M:%S ') + "Backup Canceled: Library import failed.\n")
+        backupLog.write("\n" + datetime.now().strftime('%Y-%m-%d %H:%M:%S ') +
+                        "Backup Canceled: Library import failed.\n")
     except:
-        backupLog.write("\nTIME_UNKNOWN Backup Canceled: Library import failed.\n")
+        backupLog.write(
+            "\nTIME_UNKNOWN Backup Canceled: Library import failed.\n")
     print("Backup canceled: Library import failed.")
     exit()
 
 os.chdir(os.path.dirname(os.path.abspath(__file__)))
 # creates a file called 'backup.log'
 backupLog = open(logLoc, 'a')
-backupLog.write("\n" + datetime.now().strftime('%Y-%m-%d %H:%M:%S ') + "Backup script ran.\n")
+backupLog.write(
+    "\n" + datetime.now().strftime('%Y-%m-%d %H:%M:%S ') + "Backup script ran.\n")
 
 
 # checks if the config file exists. if it doesn't, it will attempt to retrieve it from github
 
 if not os.path.exists("backupcfg.py"):
-    cfgUrl = "https://raw.githubusercontent.com/30023344/Backup-Script/d55509ceb6656491c48592b0df6ece17aff9d1f5/backupcfg.py" # doesnt work, link is not permanent 
+    # doesnt work, link is not permanent
+    cfgUrl = "https://raw.githubusercontent.com/30023344/Backup-Script/d55509ceb6656491c48592b0df6ece17aff9d1f5/backupcfg.py"
     try:
         import urllib.request
         urllib.request.urlretrieve(cfgUrl, "backupcfg.py")
-        backupLog.write(datetime.now().strftime('%Y-%m-%d %H:%M:%S ') + "'backupcfg.py' was created. Retrieved from '%s'.\n"%cfgUrl)
+        backupLog.write(datetime.now().strftime('%Y-%m-%d %H:%M:%S ') +
+                        "'backupcfg.py' was created. Retrieved from '%s'.\n" % cfgUrl)
         print("'backupcfg.py' was created.")
     except:
-        backupLog.write(datetime.now().strftime('%Y-%m-%d %H:%M:%S ') + "Backup Canceled: couldn't retrieve 'backupcfg.py' from '%s'.\n"%cfgUrl)
-        print("Backup canceled: Couldn't retrieve 'backupcfg.py' from '%s'."%cfgUrl)
+        backupLog.write(datetime.now().strftime('%Y-%m-%d %H:%M:%S ') +
+                        "Backup Canceled: couldn't retrieve 'backupcfg.py' from '%s'.\n" % cfgUrl)
+        print("Backup canceled: Couldn't retrieve 'backupcfg.py' from '%s'." % cfgUrl)
         exit()
 
```

Comment:
```suggestion
try:
    from backupcfg import *
except:
    backupLog.write("\n" + datetime.now().strftime('%Y-%m-%d %H:%M:%S ') + "Backup Cancelled: Importing config file failed. Check to see if jobs are defined correctly.\n")
```
this should hopefully fix issue #12 where incorrectly defining jobs cause a crash.